make TFLite kernel tests work

freedomtan opened the pull request:
make
```
bazel build --config opt //tensorflow/contrib/lite/kernels:all
```
work. And then
```
bazel test --config opt //tensorflow/contrib/lite/kernels:all
```
works.

freedomtan committed:
make TFLite kernel tests work

tensorflowbutler commented:
Nagging Reviewer @aselle: It has been 14 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

tensorflowbutler commented:
Nagging Reviewer @aselle: It has been 29 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

tensorflowbutler commented:
Nagging Reviewer @aselle: It has been 46 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

freedomtan committed:
add more fixes

rebase and add more fixes so that
```
bazel test --config opt //tensorflow/contrib/lite/kernels:all
```
will work
